controls/radmap/features/providers/azuremapprovider.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Azure Map Provider
```

Comment:
Is this available for Silverlight? If not, please exclude the article in the _silverlight.yml file